### PR TITLE
Tune eng process.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,21 +6,43 @@ This repository is used to track anything related to the platform that powers SU
...
 
+#### Triage column
+
+Catch-all column. Anything filed in this repository goes to this column.
+Triage should happen regularly and the column should not have many cards. A number of 20 should
+be the maximum.
+
+A card from here either moves to the backlog or it is closed.
+
+**A card can be closed, even if it is valid, if it is not going to be worked on in more than three months**.
```
Comment (LeoMcA) [inline README.md:33]: How does this interact with bugs we want to have open for community etc to work on? Perhaps we should transfer the issue into the project-specific repository?

File: README.md [hunk 2]
```diff
@@ -32,23 +54,31 @@ A description why this issue is blocked should exist in the card.
 Anything that it's actively worked on.
 Ideally nobody should own more that three cards in this column.
 
-A card is moved to the next column only when it is deployed to the testing environment.
+A card is moved to the next column only when a PR is opened.
+
+#### Review column
```
Comment (LeoMcA) [inline README.md:59]: +1

**Review by LeoMcA — APPROVED** [review]
Looks good - open question about interaction with community processes, but I don't think that has to block this.

File: README.md (hunk 1 above)
Comment (akatsoulas) [reply README.md]: We can add a label for this type of cards. I will add it in the PR.

**Pull request merged**